c2corg_ui/static/partials/feedtable.html

```diff
@@ -0,0 +1,66 @@
+<div ng-show="wfeedCtrl.documents.length > 0"
+  infinite-scroll="wfeedCtrl.getDocumentsFromWhatsnewFeed()"
+  infinite-scroll-immediate-check="false"
+  infinite-scroll-distance="0.7"
+  infinite-scroll-disabled="wfeedCtrl.busy || wfeedCtrl.error || wfeedCtrl.feedEnd || wfeedCtrl.noFeed">
+
+  <div class="table-responsive">
+    <table class="text-center feedtable table table-condensed table-striped table-hover">
+
+      <p class="text-success text-center" ng-if="wfeedCtrl.userId"
+         ng-init="user = wfeedCtrl.userId;" ng-model="amount = wfeedCtrl.feedLength;">
+        This feed for the user {{user}} contains {{amount}} changes.
+      </p>
+      <p class="text-success text-center" ng-if="!wfeedCtrl.userId"
+         ng-model="amount = wfeedCtrl.feedLength;">
+        This feed contains {{amount}} changes.
+      </p>
+
+      <tr>
+        <th class="text-center" translate>title</th>
+        <th class="text-center" translate>type</th>
+        <th class="text-center" translate>written_at</th>
+        <th class="text-center" translate>quality</th>
+        <th class="text-center" translate>username</th>
+        <th class="text-center" translate>comment</th>
+      </tr>
+
+      <tr class="feed-list"
+              ng-repeat="doc in wfeedCtrl.documents track by doc.version_id"
+          ng-init="type = wfeedCtrl.documentType(doc.document.type); console.log(type);" >
+
+        <td class="text-left">
+          <a href="{{wfeedCtrl.getDocHref(doc)}}">{{::doc['document']['title']}}</a> ({{::doc['lang']}})
+          <span ng-if="doc['document']['version'] > 1 && (doc['document']['type'] !== 'i' && doc['document']['type'] !== 'o')">
+            (<a ng-href="{{wfeedCtrl.getVersionHref(doc)}}">previous version</a> | diff)
```

Comment:
> I think that it is better to have those links (to document + to the previous version)

I am not sure to understand what you mean. Actually why do we need links towards several versions?

What do you call "previous version"? It seems your "previous version" links point to the archived version concerned by the change, doesn't it? This is actually the main link we want to see.

If you have several changes in a row of the same document, it's no use that the main links (on titles) point to the same URL...

The goal is to report that user X has created/edited a document at time T and to show what changes they have made. It should work a bit like the history pages (no need to compare several versions though) => point to the concerned version of the document and (if possible - not sure the previous version id is provided by the API response, is it?) to the diff of the changes that have been made (no diff anyway if this is a creation).
No need to point towards the last version of the document (or as a secondary link, could be named "current version").
No need to point towards the previous version of the document.

>  if it is better to have link to the document on the badge and if the document has any previous version

I don't know. In detail view pages, the badge points to the advanced search page. It would not be very consistent to make it point here to the document itself.
On the other hand, the badge name uses the singular form, not the plural form, so it's not really the same badge...